tool/gravity/cli/checks.go

```diff
@@ -134,7 +134,7 @@ func checkUpgrade(ctx context.Context, env *localenv.LocalEnvironment, config pr
 		return trace.Wrap(err)
 	}
 	// Deploy RPC agents that will be used for running checks on the nodes.
-	credentials, err := rpcAgentDeployHelper(ctx, env, "", "")
+	credentials, err := rpcAgentDeployHelper(ctx, env, "", "", "", "")
```

Comment:
Pretty hard to tell what the values are and why empty. In this case I'd suggest an options struct which would be provided to the function as empty.